static/src/javascripts/bootstraps/commercial.js

```diff
@@ -37,31 +36,27 @@ import {
 import { trackPerformance } from 'common/modules/analytics/google';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
 
-const commercialModules: Array<Array<any>> = commercialFeatures.adFree
-    ? [
-          ['cm-highMerch', highMerch.init],
-          ['cm-thirdPartyTags', thirdPartyTags.init],
-          ['cm-prepare-googletag', prepareGoogletag.init, true],
-          ['cm-closeDisabledSlots', closeDisabledSlots],
-          ['cm-fill-advert-slots', fillAdvertSlots, true],
-          ['cm-paidContainers', paidContainers],
-          ['cm-paidforBand', paidforBand.init],
-      ]
-    : [
-          ['cm-highMerch', highMerch.init],
-          ['cm-thirdPartyTags', thirdPartyTags.init],
-          ['cm-prepare-sonobi-tag', prepareSonobiTag.init, true],
-          ['cm-prepare-switch-tag', prepareSwitchTag.init, true],
-          ['cm-articleAsideAdverts', articleAsideAdvertsInit, true],
-          ['cm-prepare-googletag', prepareGoogletag.init, true],
-          ['cm-articleBodyAdverts', articleBodyAdvertsInit],
-          ['cm-liveblogAdverts', initLiveblogAdverts, true],
-          ['cm-closeDisabledSlots', closeDisabledSlots],
-          ['cm-stickyTopBanner', stickyTopBanner.init],
-          ['cm-fill-advert-slots', fillAdvertSlots, true],
-          ['cm-paidContainers', paidContainers],
-          ['cm-paidforBand', paidforBand.init],
-      ];
+const commercialModules: Array<Array<any>> = [
+    ['cm-highMerch', highMerch.init],
+    ['cm-thirdPartyTags', thirdPartyTags.init],
+    ['cm-prepare-googletag', prepareGoogletag.init, true],
+    ['cm-closeDisabledSlots', closeDisabledSlots],
+    ['cm-paidContainers', paidContainers],
+    ['cm-paidforBand', paidforBand.init],
+];
+
+if (!commercialFeatures.adFree) {
+    commercialModules.push(
```

Comment:
Doesn't this completely change the ordering of the initialisation? For example, `cm-articleAsideAdverts` will happen after 
```javascript
['cm-highMerch', highMerch.init],
['cm-thirdPartyTags', thirdPartyTags.init],
['cm-prepare-googletag', prepareGoogletag.init, true],
['cm-closeDisabledSlots', closeDisabledSlots],
['cm-paidContainers', paidContainers],
['cm-paidforBand', paidforBand.init],

['cm-prepare-sonobi-tag', prepareSonobiTag.init, true],
['cm-prepare-switch-tag', prepareSwitchTag.init, true],
```